services-toolkit/about.hbs.md

```diff
@@ -1,13 +1,40 @@
 # Services Toolkit
 
-Services Toolkit comprises the following Kubernetes-native components, which support the management,
-lifecycle, discoverability, and connectivity of service resources on Kubernetes, such as databases,
-message queues, DNS records, and more:
+Services Toolkit ("STK" for short) is responsible for backing many of the most exciting and powerful services capabilities in Tanzu Application Platform ("TAP" for short). From the integration of an endless list of cloud-based and on-prem services, through to the offering and discovery of those services, and finally to the claiming and binding of service instances to application workloads, services toolkit has the tools you need to make working with services on TAP simple, easy and effective.
 
-- Service offering
-- Service API projection
-- Service resource replication
-- Service resource claims
+## Capabilities
 
-To learn more about Services Toolkit, see the
-[Services Toolkit documentation](https://docs.vmware.com/en/Services-Toolkit-for-VMware-Tanzu-Application-Platform/0.9/svc-tlk/overview.html)
+The main capabilities on offer in TAP through services toolkit are:
+
+1. The classes and claims abstraction - providing a simple, but powerful, user experience to apps teams, while serving to promote a strong separation of concerns between apps teams and ops teams alike
```

Comment:
Suggestion: Might be worth adding - "To find out more about the classes and claims abstractions please see Four Levels of Service consumption" as a link.